gensim/models/atmodel.py

```diff
@@ -23,6 +23,9 @@
 <https://arxiv.org/abs/1207.4169>`_. The model correlates the authorship information with the topics to give a better
 insight on the subject knowledge of an author.
 
+.. _'Online Learning for LDA' by Hoffman et al.: online-lda_
```

Comment:
I think these changes are not related to this PR. Can you revert them? Alternatively, you may rebase off upstream/develop, these changes have already been merged in there.

Reply:
I messed up with Github's new fetch upstream feature and opened a new PR.